src/utils/rfq_blockchain_utils.ts

```diff
@@ -43,6 +45,8 @@ export class RfqBlockchainUtils {
     private readonly _exchangeProxy: IZeroExContract;
     private readonly _web3Wrapper: Web3Wrapper;
     private readonly _abiDecoder: AbiDecoder;
+    // An ethers.js Wallet. Must be populated for RfqBlockchainUtils instances used by RFQM Workers.
+    private readonly _ethersWallet: Wallet | undefined;
```

Comment:
is this optional only because we're in the transition phase from web3wrapper to ethers? If so, can we drop the above comment and replace it with a TODO to make this non-optional once the migration is complete? Or just some message to remind us to remove the `undefined` union type at some point.

Reply:
Hmmmm.. it's only necessary if you're using `RfqBlockchainUtils` as a Worker (the API version doesn't need to sign/submit transactions). We could make it mandatory, but then we'd have to mock it for tests where it's not used. What do you think?